### PR TITLE
Add webworldwind.org overview and features content

### LINKED ISSUE #28: Port Web Overview/Features
Transition the content from webworldwind.org to new site. Combine the overview and feature pages.

### PR DESCRIPTION
**Pull request by zglueck**
This pull request closes #28 by porting the Overview and Features content from webworldwind.org.

Most of the content is ported word for word; however, there is a special **Download** section which makes a general note of how to build the library locally as the repository is setup now in the develop branch. Additionally, I've added content which makes reference to the upcoming release and NPM inclusion.

This effort supports the #14 Port external site content epic.

### CODE REVIEW

File: content/web/_index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Get Started
+
+Web WorldWind is a 3D virtual globe API for HTML5 and JavaScript. Web pages include Web WorldWind to provide one or more virtual globes on that page. The [European Space Agency](https://www.esa.int/) is now partnering with NASA on development. Work on Web WorldWind began in 2014 and is very active today.
+
+---
+
+## Download
+
+*The first release of Web WorldWind is imminent, please check back soon for updates to these instructions! The versioned release will be the first official release of Web WorldWind and will be available as a hosted library and through NPM.*
```
Comment (pdavidc) [inline content/web/_index.md:20]: This formatting makes sense conceptually, but looked odd to me.

File: content/web/_index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+To add Web WorldWind to your web application, build the worldwind.js and worldwind.min.js files locally and host the the library in a manner which best supports your applications requirements. To build the worldwind.js and worldwind.min.js files in your own environment follow the directions in the [HowToBuildWebWW](https://github.com/NASAWorldWind/WebWorldWind/blob/develop/HowToBuildWebWW.md) document in the repository.
+
+For more information about integrating Web WorldWind with your site and for how to use its many features, please see the [Developer Guide](/web/developer-guide/).
```
Comment (pdavidc) [inline content/web/_index.md:24]: Did you mean tutorials?

File: content/web/_index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Web WorldWind is a 3D virtual globe API for HTML5 and JavaScript. Web pages include Web WorldWind to provide one or more virtual globes on that page. The [European Space Agency](https://www.esa.int/) is now partnering with NASA on development. Work on Web WorldWind began in 2014 and is very active today.
+
+---
+
+## Download
+
+*The first release of Web WorldWind is imminent, please check back soon for updates to these instructions! The versioned release will be the first official release of Web WorldWind and will be available as a hosted library and through NPM.*
+
+To add Web WorldWind to your web application, build the worldwind.js and worldwind.min.js files locally and host the the library in a manner which best supports your applications requirements. To build the worldwind.js and worldwind.min.js files in your own environment follow the directions in the [HowToBuildWebWW](https://github.com/NASAWorldWind/WebWorldWind/blob/develop/HowToBuildWebWW.md) document in the repository.
```
Comment (pdavidc) [inline content/web/_index.md:22]: Could this be expressed as a simple list? It's wordy and difficult to make out what actions are needed. The result sounds a lot more complex than it is.

**Review by pdavidc — CHANGES_REQUESTED** [review]
Very nice beginning for Web WorldWind. Just a few changes needed.

**Comment by zglueck** [comment]
I'm closing this pull request in favor of #49 which incorporates some site architecture changes merged in other pull requests.

**Pull request closed without merging**